pymultimonaprs.json

```diff
@@ -9,7 +9,8 @@
 		"ppm": 0,
 		"gain": 0,
 		"offset_tuning": false,
-		"device_index": 0
+		"device_index": 0,
+		"debug": 0
```

Comment:
- Wrong config section - this is not specific to the rtl backend
- Wrong value type - use bool true/false

Also I would call the key `verbose` to use the same name as for the cli argument